community/modules/compute/schedmd-slurm-gcp-v6-nodeset/variables.tf

```diff
@@ -547,7 +547,7 @@ variable "maintenance_interval" {
 variable "startup_script" {
   description = "Startup script used by VMs in this nodeset"
   type        = string
-  default     = null
+  default     = "# no-op"
```

Comment:
Let's rebase, shoudn't be needed anymore

Reply:
Rebased off current develop